variables.tf

```
@@ -87,12 +87,23 @@ variable "docker_image" {
   default     = "prestosql/presto:341"
 }
 
+variable "consul_connect_plugin" {
+  type = bool
+  description = "Deploy consul connect plugin for presto"
+  default = true
+}
+
 variable "consul_connect_plugin_version" {
   type = string
   description = "Version of the consul connect plugin for presto (on maven central) src here: https://github.com/gugalnikov/presto-consul-connect"
   default = "2.2.0"
 }
 
+variable "consul_connect_plugin_artifact_source" {
+  type = string
+  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect/"
```

host address is not correct
```suggestion
  default = "https://oss.sonatype.org/service/local/repositories/releases/content/io/github/gugalnikov/presto-consul-connect"
```